devel/libuv/files/patch-libuv-legacy-devel.diff

```diff
@@ -31,7 +31,17 @@ diff --git src/unix/fs.c src/unix/fs.c
 index 6b051c12..18e274cf 100644
 --- src/unix/fs.c
 +++ src/unix/fs.c
-@@ -1410,7 +1410,7 @@ static void uv__to_stat(struct stat* src, uv_stat_t* dst) {
+@@ -84,7 +84,8 @@
+ 
+ #if defined(__CYGWIN__) ||                                                    \
+     (defined(__HAIKU__) && B_HAIKU_VERSION < B_HAIKU_VERSION_1_PRE_BETA_5) || \
+-    (defined(__sun) && !defined(__illumos__))
++    (defined(__sun) && !defined(__illumos__)) ||                              \
++    (defined(__APPLE__) && MAC_OS_X_VERSION_MIN_REQUIRED <= 101300)
```

Comment:
Why "less than or equal to 10.13.0"? Surely you mean either "less than 10.13.0" or "less than 10.14.0" unless you really do want to handle 10.13.0 and earlier differently from 10.13.1 and later.

Reply:
@ryandesign Are these macros defined also for microversions?

Reply:
> @ryandesign Are these macros defined also for microversions?

https://github.com/kencu/macos-sdk/blob/6c1513f5b0667b76e24aaadcad130e90c545f046/MacOSX14.0.sdk/usr/include/AvailabilityVersions.h#L27

Reply:
Yeah, then this should be changed so that all 10.13.x are treated equally.